pkg/operator/k8sutil/node.go

```diff
@@ -299,6 +296,33 @@ func GetKubernetesNodesMatchingRookNodes(rookNodes []rookv1.Node, clientset kube
 	return nodes, nil
 }
 
+// GetKubernetesNodes lists and returns all the nodes in Kubernetes.
+func GetKubernetesNodes(clientset kubernetes.Interface) (*v1.NodeList, error) {
+	ctx := context.TODO()
+	nodeOptions := metav1.ListOptions{}
+	nodeOptions.TypeMeta.Kind = "Node"
```

Comment:
Isn't everything returned by `Nodes().List()` of `kind: Node`? This doesn't seem necessary. I suggest we don't need this `GetKubernetesNodes()` helper. Instead, calling `Nodes().List()` should work fine.

Reply:
removed `GetKubernetesNodes()` helper